spec/system/community_sidebar_section_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+describe "Community sidebar section", type: :system do
```

Comment:
I think we can just merge this test case into the `viewing_sidebar_spec.rb` system test file.